index.html

```diff
@@ -244,53 +286,58 @@ <h3 class="skill-img ab"> <img src="img/framwork56.png" alt=""> Framework</h3>
         </div>
         <div class="card-skill">
             
-            <h3 class="skill-img"> <img src="img/round57.png" alt=""> Skill</h3>
-            <ul class="skill-group">
+            <h3 class="skill-img rab"> <img src="img/round57.png" alt=""> Skill</h3>
```

Comment:
- [Optional] please make sure that the buttons are matching the design, there is a weird border around these buttons also make sure to provide hovering styles.

| Yours  | Figmas |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/215293821-f1fc394b-d60e-4360-8a7b-5c830d8ba264.png) | ![image](https://user-images.githubusercontent.com/74744787/215293840-96142f98-e662-4f7c-950d-fb840955e3f5.png)  |